src/protocols/raydium.ts

```diff
@@ -28,6 +28,7 @@ import {
 } from "../ids";
 import { Gateway } from "@dappio-wonderland/gateway-idls";
 import { PoolInfoWrapper } from "@dappio-wonderland/navigator/dist/raydium";
```

Comment:
This import is redundant. You can access `PoolInfoWrapper` via `raydium.PoolInfoWrapper`